e2e/fixtures/dotnet/sdk/Project.csproj

```diff
@@ -24,4 +24,31 @@
     <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- @OctoLinkerResolve(https://www.nuget.org/packages/ClassicProject.csproj) -->
+    <ProjectReference Include="../Classic/Project.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <!-- @OctoLinkerResolve(Directory.Build.props) -->
+    <Compile Include="../Directory.Build.props" />
```

Comment:
@stefanbuck what value should these resolve to? The test results seem to be pointing to nuget.org instead of github.com.

Reply:
If these links are meant to be relative the annotation needs to be 
`// @OctoLinkerResolve(<root>/dotnet/Directory.Build.props)`

`<root>` will be replace with `https://github.com/OctoLinker/OctoLinker/tree/e2e/fixtures` when generate the test data out of all fixtures. 

However, this does not explain why the test is pointing to `nuget.org`.

Reply:
Strange, I checkout out this PR and hooked up everything locally. Navigating to https://github.com/OctoLinker/OctoLinker/blob/94beb901c9310f2d5c9d431e5ad76cb67c87a1c3/e2e/fixtures/dotnet/sdk/Project.csproj#L29 does not link this line at all.

Reply:
Well the casing of `/Classic/` is different on disk. However, the next include does not work either https://github.com/OctoLinker/OctoLinker/blob/94beb901c9310f2d5c9d431e5ad76cb67c87a1c3/e2e/fixtures/dotnet/sdk/Project.csproj#L34

Reply:
The path casing was wrong. `Classic` should have been all lowercase. Switching that fixes it.

Reply:
The tests are passing now for these, but some other ones are failing that aren't related to this change 😕

Reply:
For some odd reason, E2E tests are timing out way more often on PRs than on [master](https://github.com/OctoLinker/OctoLinker/actions?query=branch%3Amaster). Let's get this merged and we will see what happens 😉

Reply:
It's passing 😄